campaigns.json

```diff
@@ -1331,9 +1331,9 @@
       }
     },
    "SiteBlacklist": [
-     "T2_CH_CERN",
-     "T2_CH_CERN_HLT"
-         ], 
+         "T2_CH_CERN",
```

Comment:
improper indentation

Reply:
I think you mean here this was an old change that we agreed would be reformatted when we reformatted everything:
https://github.com/scarletnorberg/WmAgentScripts/blob/June9th/campaigns.json#L1321-L1353

Reply:
I will make a new PR with reformatting.